src/common/io.py

```diff
@@ -2757,10 +2759,9 @@ def diagnostics_point(self, diag_data):
         """ Generates a data point for diagnostics data by invoking the util function save_diagnostics_point. """
         self.dump_data_internal.save_diagnostics_point(diag_data)
         self.nbr_diag_points += 1
-        self._make_diagnostics_consistent()
+        self._make_consistent(diag=True)
 
-    def _make_consistent(self):
-        """ TODO Requires documentation. """
+    def _make_consistent(self, diag=False):
```

Comment:
Can we add a docstring to what this function does? In general I think a lot of functions in PyFMI require some documentation, in order to improve understanding.

Reply:
Added.